cmake/modules/RootMacros.cmake

```diff
@@ -148,7 +148,7 @@ macro(REFLEX_GENERATE_DICTIONARY dictionary)
   set(gensrcdict ${dictionary}.cxx)
 
   #---roottest compability---------------------------------
-  if(CMAKE_ROOTTEST_NOROOTMAP)
+  if(CMAKE_ROOTTEST_NOROOTMAP OR cpp_module_file)
```

Comment:
Do we still need `CMAKE_ROOTTEST_NOROOTMAP`? Weren't "you" the only "customer" of that?

Reply:
If “you” means modules support then no. Some amount of tests are using it and I think they were the reasons to invent it. It is a misnomer because rootcling still produces “.out” files to aid aclic but that’s another story.

Reply:
And, this was a merge failure which actually needs to be moved in ROOT_GENERATE_DICTIONARY...